modules/apps/change-tracking/change-tracking-change-lists-configuration-web/src/main/java/com/liferay/change/tracking/change/lists/configuration/web/internal/configuration/ChangeListsConfiguration.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.change.tracking.change.lists.configuration.web.internal.configuration;
+
+import aQute.bnd.annotation.ProviderType;
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author David Truong
+ */
+@ExtendedObjectClassDefinition(category = "change-lists")
```

Comment:
Should this be scoped to SYSTEM or COMPANY?  I'd assume we want to turn this on/off by company?